R/calcFEdemand.R

```diff
@@ -1,17 +1,16 @@
 #' Calculates Final Energy Demand for Industry and Buildings
+#' @param scenario GDP and pop scenarios. Passed to [mrdrivers::calcGDP()].
 #' @author Falk Benke
-calcFEdemand <- function() {
+calcFEdemand <- function(scenario) {
 
-  feBuildings <- calcOutput("FeDemandBuildings", subtype = "FE", warnNA = FALSE, aggregate = FALSE)
-  feIndustry <- calcOutput("FeDemandIndustry", warnNA = FALSE, aggregate = FALSE)
+  feBuildings <- calcOutput("FeDemandBuildings",
+                            subtype = "FE",
+                            scenario = scenario,
+                            warnNA = FALSE,
+                            aggregate = FALSE)
+  feIndustry <- calcOutput("FeDemandIndustry",  scenario = scenario, warnNA = FALSE, aggregate = FALSE)
```

Comment:
@LaviniaBaumstark @0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q @johanneskoch94 This should be `scenarios` as an argument, not `scenario`, right? This should explain the problems with caching we are experiencing currently.

Reply:
https://github.com/pik-piam/mrremind/pull/642

Reply:
@johanneskoch94 I was in a rush to fix this. Maybe you want to solve this differently by rather renaming the argument in `calcFeDemandIndustry` to `scenario` for improved consistency.

Reply:
Fine by me, just be aware that it is `scenario` in mrremind - and madrat won't necessarily tell you if you get it wrong :)